pages/common/pax.md

```diff
@@ -0,0 +1,27 @@
+# pax
+
+> Archiving and copying utility.
```

Comment:
Perhaps we could add a line after this one for the homepage of this program, like this?

```
> Homepage: <INSERT_URL_HERE>.
```

Reply:
I'm afraid I couldn't really find a homepage. `pax` is a standard core utility, that is defined by the POSIX.1-2001 standard and there are different implementations out there. Should I link to the [entry in the POSIX standard](http://pubs.opengroup.org/onlinepubs/009695399/utilities/pax.html)?

Reply:
No need if it's a core utility.